automation/WinFormsAutomationApp/MainForm.cs

```diff
@@ -36,11 +36,11 @@ void LogCallback(LogLevel level, string message, bool containsPii)
 
         private void acquireToken_Click(object sender, EventArgs e)
         {
-            _commandToRun = AuthenticationHelper.AcquireToken;
+            _commandToRun = AuthenticationHelper.AcquireTokenAsync;
             pageControl1.SelectedTab = dataInputPage;
         }
 
-        private async void requestGo_Click(object sender, EventArgs e)
+        private async void RequestGo_Click(object sender, EventArgs e)
```

Comment:
>RequestGo [](start = 27, length = 9)

minor: any reason for this change? the field seems to be called requestGo, thus perhaps it is not needed to change the method name?

Reply:
it was a suggestion by the code analysis, not really needed actually, ill revert